src/renderer/user-styles-component.ts

```diff
@@ -24,8 +24,12 @@ export const userStylesComponent = Vue.extend({
 
             --user-input--background-color: {{ colortheme.userInputBackgroundColor }};
             --user-input--color: {{ colortheme.userInputTextColor }};
+            --user-input--radius:{{appearance.userInputRadius}};
+            --user-input--bottom-margin:{{appearance.userInputBottomMargin}}px;
```

Comment:
Generally I suggest adding a space between the css property and value, and also when defining variables. I think it's easier to read.